sensing/position_estimation.c

```diff
@@ -212,49 +209,60 @@ static void position_estimation_position_correction(position_estimation_t *pos_e
 			local_coordinates = coord_conventions_global_to_local_position(global_gps_position,pos_est->local_position.origin);
 			local_coordinates.timestamp_ms = pos_est->gps->time_last_msg;
 			
-			// compute GPS velocity estimate
-			gps_dt = (local_coordinates.timestamp_ms - pos_est->last_gps_pos.timestamp_ms) / 1000.0f;
-			if (gps_dt > 0.001f)
-			{
-				for (i = 0; i < 3; i++)
-				{
-					pos_est->last_vel[i] = (local_coordinates.pos[i] - pos_est->last_gps_pos.pos[i]) / gps_dt;
-				}
-				pos_est->last_gps_pos = local_coordinates;
-			}
-			else
-			{
-				print_util_dbg_print("GPS dt is too small!\r\n");
-			}
+			pos_est->last_gps_pos = local_coordinates;
 		}
-		t_inter_gps = time_keeper_get_millis() - pos_est->gps->time_last_msg;
-			
-		//gps_gain = maths_f_max(1.0f - t_inter_gps / 1000.0f, 0.0f);
+		
```

Comment:
gps_update = 4Hz
position_estimation_update = 250Hz
Doing the correction even with older value gives a better estimation, correcting slightly at every time step the integration of the accelerometers.